src/main/java/hw6/VehicleTest.java

```diff
@@ -0,0 +1,57 @@
+class Vehicle {
```

Comment:
Move all these classes as a separate package class please. Fix it in all places please
![Screenshot_1](https://github.com/user-attachments/assets/8f8a8f75-7c10-4e45-bf7a-f299071b053d)